package.json

```diff
@@ -3,11 +3,11 @@
   "version": "0.0.0",
   "license": "MIT AND Apache-2.0",
   "scripts": {
-    "build": "turbo build --filter='!@recallnet/api-sdk'",
+    "build": "turbo build",
     "dev": "turbo dev --concurrency=20",
     "lint": "turbo lint",
-    "format": "prettier --write \"**/*.{ts,tsx,md,json}\"",
-    "format:check": "prettier --check \"**/*.{ts,tsx,md}\"",
+    "format": "turbo format",
```

Comment:
e.g., this matches vercel's [turborepo script](https://github.com/vercel/turborepo/blob/main/package.json#L9).